test/cases/compile_errors/duplicate_union_field.zig

```diff
@@ -11,6 +11,6 @@ export fn entry() void {
 // backend=stage2
 // target=native
 //
-// :3:5: error: duplicate union field: 'Bar'
-// :2:5: note: other field here
+// :2:5: error: duplicate union field: 'Bar'
+// :3:5: note: other field here
```

Comment:
I don't like that it's reversed the order of which one shows up as the duplicate

Reply:
The order might be reversed but it makes more sense to list the duplicates top to bottom now that there can be multiple.

Reply:
maybe `union field name conflict: 'Bar'` ?

Reply:
something like:
```
:2:5: error: union field name conflict: 'Bar'
:3:5: note: duplicate field here
```

Reply:
after some thought i agree with this... the current error messages are a backwards. 

ill change the order around 👍